src/io/http/src/main/scala/services/TextAnalytics.scala

```diff
@@ -218,7 +218,18 @@ class EntityDetector(override val uid: String)
 
   def setLocation(v: String): this.type =
     setUrl(s"https://$v.api.cognitive.microsoft.com/text/analytics/v2.0/entities")
+}
+
+object CloudNER extends ComplexParamsReadable[CloudNER]
 
+class CloudNER(override val uid: String) extends TextAnalyticsBase(uid) {
```

Comment:
Nit: I think it might make sense to rename the other skill to LocalNER, and rename this back to regular NER. Sorry for the randomization